axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -733,25 +833,43 @@
             "10"
           ]
         },
-        "codeId": 737,
-        "lastUploadedCodeId": 737,
-        "address": "axelar1vaj9sfzc3z0gpel90wu4ljutncutv0wuhvvwfsh30rqxq422z89qnd989l"
-      },
-      "NexusGateway": {
-        "nexus": "axelar17h8uk4ct0mdv9mgkuxszt4gp2htpfr08mge20r",
-        "codeId": 619,
-        "lastUploadedCodeId": 619,
-        "address": "axelar1jjjr3tqs0nzjv3y9fg4xvzkww50jq06a9qp77r8kzmqyla97556sxx7702"
+        "codeId": 849,
+        "lastUploadedCodeId": 849,
+        "address": "axelar1vaj9sfzc3z0gpel90wu4ljutncutv0wuhvvwfsh30rqxq422z89qnd989l",
+        "axelar": {
+          "codeId": 849
+        },
+        "storeCodeProposalId": "69",
+        "storeCodeProposalCodeHash": "095c1caca4f9b7381519bd8395f3f558202fd4d4ad03f223dd8a2e991c568bd6"
       },
       "AxelarnetGateway": {
-        "codeId": 661,
-        "address": "axelar1yvfcrdke7fasxfaxx2r706h7h85rnk3w68cc5f4fkmafz5j755ssl8h9p0"
+        "codeId": 844,
+        "address": "axelar1wvms3cy5hxrgl7uxhkz7yth4qzqum6aaccwkmvafq8z0mgdfxr8qrnvw0k",
+        "nexus": "axelar17h8uk4ct0mdv9mgkuxszt4gp2htpfr08mge20r",
+        "axelar": {
+          "storeInstantiateProposalId": "59",
+          "codeId": 845
+        },
+        "storeCodeProposalCodeHash": "c7286d0f59276b794641bdfbb4f96fafcee3553b67f3397d662a4683968f525b",
+        "storeCodeProposalId": "60",
+        "lastUploadedCodeId": 845
       },
       "InterchainTokenService": {
         "adminAddress": "axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9",
         "governanceAddress": "axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9",
-        "codeId": 738,
-        "address": "axelar10jzzmv5m7da7dn2xsfac0yqe7zamy34uedx3e28laq0p6f3f8dzqp649fp"
+        "codeId": 843,
+        "address": "axelar157hl7gpuknjmhtac2qnphuazv2yerfagva7lsu9vuj2pgn32z22qa26dk4",
+        "axelar": {
+          "codeId": 843,
+          "instantiateProposalId": "61"
+        },
+        "sui-test2": {
+          "maxUintBits": 64,
+          "maxDecimalsWhenTruncating": 6
+        },
```

Comment:
why is sui included here?

Reply:
This is required for the submit proposal script to Register ITS contract on ITS Hub
 @eguajardo

Reply:
why would sui be required, you need to register the EVM chains as part of this